### DISCUSSION TITLE
Open Calls

### DISCUSSION THREAD

**Discussion by shiftbug** [daveshap/ACE_Framework#29] [General]
Could we schedule some well moderated open video calls with one purpose among others, in particular naturally clustering parties interested in participating. The idea being that the core team openly discussing current status and pursuits (maybe with live public engagement via chat) could provide talking points around which interested parties could gravitate and distill into peripheral teams.

Hi, I’m Shiftbug, an llm/python noob, thinker, gamer. Former  Registered Behavioral Technician, Civil Designer, Cold Sales Rep. Working on schema driven llm production. Obsessed with creative abstraction, behavior modeling, and human flourishing.

**Comment by daveshap** [comment]
That's a fantastic idea and my initial thought is "probably yes" but we are not quite ready for that level of community engagement. Stay tuned... 

@lancecarlson
